src/Catalyst.Core.Modules.Kvm/DeltaExecutor.cs

```diff
@@ -99,33 +100,30 @@ public sealed class DeltaExecutor : IDeltaExecutor
         /// <param name="entry"></param>
         /// <param name="releaseSpec"></param>
         /// <returns>Total intrinsic cost of the <paramref name="entry" /></returns>
-        public static ulong CalculateIntrinsicGas(ContractEntry entry, IReleaseSpec releaseSpec)
+        public static ulong CalculateIntrinsicGas(PublicEntry entry, IReleaseSpec releaseSpec)
         {
             ulong result = GasCostOf.Transaction; // the basic entry cost
             if (entry.Data != null)
             {
                 // here is the difference between the 0 bytes and non-zero bytes cost
                 // justified by a better compression level of zero bytes
-                long txDataNonZeroGasCost = releaseSpec.IsEip2028Enabled ? GasCostOf.TxDataNonZeroEip2028 : GasCostOf.TxDataNonZero;
-                int length = entry.Data.Length;
-                for (int i = 0; i < length; i++)
-                {
+                var txDataNonZeroGasCost = releaseSpec.IsEip2028Enabled
+                    ? GasCostOf.TxDataNonZeroEip2028
+                    : GasCostOf.TxDataNonZero;
+                var length = entry.Data.Length;
+                for (var i = 0; i < length; i++)
                     result += entry.Data[i] == 0 ? GasCostOf.TxDataZero : (ulong) txDataNonZeroGasCost;
-                }
             }
 
-            if (entry.IsValidDeploymentEntry && releaseSpec.IsEip2Enabled)
-            {
-                result += GasCostOf.TxCreate;
-            }
+            if (entry.IsValidDeploymentEntry && releaseSpec.IsEip2Enabled) result += GasCostOf.TxCreate;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4614986)